packages/terra-consumer-nav/src/_text_utility.scss

```diff
@@ -0,0 +1,29 @@
+@import './variables';
```

Comment:
this file should be removed. We should use TerraIcon to build our icons and that will let us delete this code.